.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -11,8 +11,19 @@ assignees: ''
 
 ### Which Crates to Increment
 
-To check if any of the top-level crates need version increments, go to the zebra GitHub code page: https://github.com/ZcashFoundation/zebra and use the last modified dates of each crate. Alternatively you can use the github compare tool and check the `main` branch against the last tag ([Example](https://github.com/ZcashFoundation/zebra/compare/v1.0.0-alpha.15...main)). `git diff --stat <previous-release-tag> origin/main` is also useful to see what's changed.
-
+To check if any of the top-level crates need version increments:
+1. go to the zebra GitHub code page: https://github.com/ZcashFoundation/zebra
```

Comment:
There are list items that start with capital letters and others do not. Do you think we should start all list items of the template with a capital letter (where possible) ?

Reply:
Thanks, fixed in c8cb7a300672c0f1899216e18dac49d297831c5e